src/components/NavBar.js

```diff
@@ -9,17 +9,11 @@ class NavBar extends React.Component {
   }
 
   render() {
-    if (!this.props.showSplash) {
+    if (!this.props.showSplashPage) {
       return (
         <nav className="NavBar">
           <Hamburger />
-          <SearchBar
-            toggleResults={this.props.toggleResults}
-            updateResults={this.props.updateResults}
-            selectResult={this.props.selectResult}
-            searchResults={this.props.searchResults}
-            showSuggestions={this.props.showSuggestions}
-          />
+          <SearchBar {...this.props} />
```

Comment:
I was thinking about that as well, I don't think it would be expensive to use the spread operator because props is an object and we are (I assume) passing references to certain parts of that object across multiple components. It's an interesting thought, I wonder if its accurate.